### PR TITLE
Esas/provide slider component 10462

### PR DESCRIPTION
**Pull request by esasova**

### CODE REVIEW

File: src/inputs/BasicInputs/BasicSliderInput/BasicSliderInput.js [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+      containerProps={containerProps}
+      labelProps={labelProps}
+      {...otherProps}
+    >
+      {disabled ? (
+        <TextField value={value} disabled />
+      ) : (
+        <Slider
+          value={value}
+          style={sliderProps}
```
Comment (csm-thu) [inline src/inputs/BasicInputs/BasicSliderInput/BasicSliderInput.js:53]: `sliderProps` sounds too generic for the "style" prop of the Slider, maybe we could rename this prop to `sliderStyle` ?

File: src/inputs/BasicInputs/BasicSliderInput/BasicSliderInput.js [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+          color={color}
+          onChange={(event, newValue) => handleSliderValueChange(parseFloat(newValue))}
+          aria-labelledby="basic-slider-input-label"
```
Comment (csm-thu) [inline src/inputs/BasicInputs/BasicSliderInput/BasicSliderInput.js:56]: `aria-labelledby` value should be "_The id of the element containing a label for the slider_", but the label associated to the slider is defined by `BasicInputWrapper`. If we don't force the `id` of the label in `BasicInputWrapper`, we should probably not define `aria-labelledby` here.

**Review by csm-thu — CHANGES_REQUESTED** [review]

**Review by nborde-CSM — APPROVED** [review]
LGTM :+1:

**Review by ardvi — APPROVED** [review]
LGTM

**Review by csm-thu — APPROVED** [review]
LGTM :+1:

**Pull request merged**